external-scripts.json

```diff
@@ -1 +1 @@
-["hubot-diagnostics", "hubot-hello", "hubot-help", "hubot-pugme", "hubot-redis-brain", "hubot-rules"]
```

Comment:
Can you also remove `hubot-pugme` from `package.json`? You might need to get rid of it from the `package-lock.json` too. 

The best way to make sure that it's gone is to `rm -f package-lock.json`, remove `hubot-pugme` from `package.json` and the run `npm install` to regenerate the package.json. 

Makes 100% sure that the package is gone from the lock file.